keyboards/converter/att56k_usb/att56k_if.c

```diff
@@ -0,0 +1,362 @@
+/*
```

Comment:
This file uses a lot of AVR-specific register definitions and the like. I do understand that the interrupts are AVR-specific, and at this stage we don't have a good abstraction over the top of these.
However, in general we'd prefer to see the GPIO operations converted over to use the platform-agnostic APIs, such as `setPinOutput()` and `writePin()` instead.